src/Tester/Tester.csproj

```diff
@@ -61,6 +61,7 @@
     <Reference Include="FluentAssertions.Core">
       <HintPath>..\packages\FluentAssertions.4.0.0\lib\net45\FluentAssertions.Core.dll</HintPath>
     </Reference>
+    <Reference Include="FSharp.Core, Version=4.4.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
```

Comment:
FSharp.Core should be sufficient here, not the full assembly version string. 

The Option stuff is fairly core F# functionality, so does not need v4.4 specifically.

FWIW, GacUtil says my machine with VS2013 installed has `FSharp.Core, Version=4.3.1.0` so don't think this change will compile in VS2013.